code/controllers/configuration/entries/general.dm

```diff
@@ -352,6 +352,14 @@
 
 /datum/config_entry/flag/check_randomizer
 
+/datum/config_entry/string/md5_salt
+	protection = CONFIG_ENTRY_HIDDEN
```

Comment:
```suggestion
	protection = CONFIG_ENTRY_LOCKED | CONFIG_ENTRY_HIDDEN
```

I know this shouldn't matter but humor me. Plus it's what the DB config uses.

Reply:
the config entry has @ at the front so it'll do this anyway 😭